docs/quickstart/4-explore-sourced.md

```diff
@@ -0,0 +1,81 @@
+# Explore source{d} CE Web Interface
+
+Once **source{d} CE** has been [initialized with `sourced init`](./3-init-sourced.md), it will be automatically opened the web UI. If the UI is not automatically opened, you can use `sourced web` command, or visit http://localhost:8088.
```

Comment:
```suggestion
Once **source{d} CE** has been [initialized with `sourced init`](./3-init-sourced.md), it will automatically open the web UI. If the UI is not automatically opened, you can use `sourced web` command, or visit http://localhost:8088.
```